libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -94,6 +94,28 @@ string IRGeneratorForStatements::code() const
 	return m_code.str();
 }
 
+bool IRGeneratorForStatements::visit(VariableDeclaration const& _varDecl)
```

Comment:
Ah sorry, I think this was a misunderstanding. I meant adding a function to IRGeneratorForStatements to generate code to initialize a single state variable. We already have `IRGeneratorForStatements::endVisit(VariableDeclarationStatement)` so the could would be duplicated for local variables.

Reply:
And perhaps to further clarify: I meant a _named_ function.